packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -95,7 +96,7 @@ export interface EditableTree extends Iterable<EditableField> {
     /**
      * Value stored on this node.
```

Comment:
Would be good to update the comment to say that the edit semantics is last-write-wins here.

Reply:
done